### DISCUSSION TITLE
why i use message.content it will return null for me ??

### DISCUSSION THREAD

**Discussion by mansan1230** [Rapptz/discord.py#7708] [Q&A] [answered]
how can i get the content of a message ??
i try to use message.content is return null for me

**Comment by SebbyLaw** [comment, marked as answer]
When using current builds of version 2.0, you must manually opt into the message content intent:
```py
intents = discord.Intents.default()
intents.message_content = True

bot = commands.Bot(..., intents=intents)
```
Or similar is required.

**Reply by mansan1230** [reply]
![image](https://user-images.githubusercontent.com/84826972/158905144-3f3f033e-a051-4d08-9b5f-abb075a2dee0.png)
i want to get the content, how can i do

**Reply by joannx** [reply]
``message.content`` is the correct way, however you need to enable the intent above. you must also explicitly enable it in the bot tab under your application in the developer portal
